### PR TITLE
Hunterized libwebp

### PR DESCRIPTION
**Pull request by mean-ui-thread**

### CODE REVIEW

File: cmake/WebPConfig.cmake.in [hunk 1]
```diff
@@ -1,6 +1,4 @@
 @PACKAGE_INIT@
 
-set(WebP_INCLUDE_DIRS "webp")
-set(WEBP_INCLUDE_DIRS ${WebP_INCLUDE_DIRS})
-set(WebP_LIBRARIES "@INSTALLED_LIBRARIES@")
-set(WEBP_LIBRARIES "${WebP_LIBRARIES}")
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
```
Comment (ruslo) [inline cmake/WebPConfig.cmake.in]: I think we need `find_package` for dependencies here, example:
* https://github.com/elucideye/drishti/blob/2ca66a7e0c92d1c1c382774b5f1ff9c34e192635/cmake/Config.cmake.in#L3-L7

Functionality similar to `find_dependency` macro but `find_dependency` will not work with `CONFIG` in CMake 3.0:
* https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

**Review by ruslo — CHANGES_REQUESTED** [review]

File: cmake/WebPConfig.cmake.in (hunk 1 above)
Comment (mean-ui-thread) [reply cmake/WebPConfig.cmake.in]: @ruslo : fixed.

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Thanks, released: https://github.com/hunter-packages/libwebp/releases/tag/v0.6.1-p0

**Comment by mean-ui-thread** [comment]
@ruslo Sweet! and thank you for creating the release as well! I will update my hunter PR right away

**Pull request merged**